### PR TITLE
Add github action to generate single API spec

### PR DESCRIPTION
**Pull request by dvacca-onfido**
Initial version of Github Action workflow to build and push single-file OpenAPI specification.

Include validation of multi file specification and a few files which were forgotten in [PR#42](https://github.com/onfido/onfido-openapi-spec/pull/42).

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -5,6 +5,7 @@ info:
   version: 3.6.0
   contact:
     name: Onfido
+    url: https://onfido.com
     email: engineering@onfido.com
```
Comment (DavidMealha-Onfido) [inline openapi.yaml:9]: This should be support's email actually

Comment (dvacca-onfido) [reply openapi.yaml]: Updated

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Onfido OpenAPI specification (beta)
 
-:warning: Our OpenAPI specification is currently in beta and we welcome any feedback. You can contact us via the issues tab or [email](mailto:openapi-feedback@onfido.com), but we don't yet officially support this specification. :warning:
+:warning: Our OpenAPI specification is currently in beta and we welcome any feedback. You can contact us via the issues tab or [support portal](https://public.support.onfido.com), but we don't yet officially support this specification. :warning:
```
Comment (DavidMealha-Onfido) [inline README.md:3]: Should we consider that the spec is out of beta since we have all the reports ?

Comment (dvacca-onfido) [reply README.md:3]: I think we should wait a little bit longer, to have one client library sufficiently tested.

**Review by RiccardoPetteruti-Onfido — APPROVED** [review]

**Review by DavidMealha-Onfido — APPROVED** [review]

**Pull request merged**